monitor.js

```diff
@@ -2,15 +2,24 @@
  * Monitor
  * Pokemon Showdown - http://pokemonshowdown.com/
  *
- * Various utility functions to make sure PS is running healthy.
+ * Various utility functions to make sure PS is running healthily.
  *
  * @license MIT license
  */
 'use strict';
 
 const FS = require('./fs');
 
+/**
+ * @class TimedCounter
```

Comment:
`@property` does work (apart from arrays), but checking [Typescript's JSDoc support](https://github.com/Microsoft/TypeScript/wiki/JSDoc-support-in-JavaScript) they don't explicitly mention that the declaration's supported. I'll update the rest of the Typescript I've written to reflect this in another pullreq